src/main/java/seedu/address/logic/parser/ParserUtil.java

```
@@ -54,7 +54,12 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         LinkedList<Index> indices = new LinkedList<>();
         for (String number : numbers) {
             number = number.trim();
-            indices.add(parseIndex(number));
+            Index indexToAdd = parseIndex(number);
+            // ignore duplicate indexes
+            if (indices.contains(indexToAdd)) {
+                continue;
```

I think you should include in the userguide that if duplicate indices are entered, the application will silently ignore the duplicated index.